packages/edit-widgets/src/components/secondary-sidebar/inserter-sidebar.js

```diff
@@ -29,19 +26,11 @@ export default function InserterSidebar() {
 	}, [ setIsInserterOpened ] );
 
 	const TagName = ! isMobileViewport ? VisuallyHidden : 'div';
-	const [ inserterDialogRef, inserterDialogProps ] = useDialog( {
-		onClose: closeInserter,
-		focusOnMount: true,
```

Comment:
We still need the `focusOnMount` or something similar to move focus to the inserter when it's opened. Otherwise focus stays on the appender after opening the inserter.

Reply:
Isn't that issue resolved by #67754? It seems to work fine in the Widget Editor too.

https://github.com/user-attachments/assets/6911f030-9657-4dd0-ba76-4946bb747f02

Reply:
Yes, sorry. I wrote that comment before I tried out https://github.com/WordPress/gutenberg/pull/67754.